test/games/strategy/triplea/delegate/GameDataTestUtil.java

```diff
@@ -80,6 +80,27 @@ public static UnitType carrier(final GameData data) {
     return unitType(Constants.UNIT_TYPE_CARRIER, data);
   }
 
+  // Some units hard coded here rather than placed in Constants at the insistence of community members Ref: Pull Request 1074
```

Comment:
Just a bit passive aggressive?
This is otherwise a very distracting comment and is unnecessary.